### PR TITLE
LPS-139802 Make AutoFields add/remove buttons accessible

### PR DESCRIPTION
**Pull request by tinycarol**
### Issues

- [LPS-139802](https://issues.liferay.com/browse/LPS-139802)

### What is the goal?

Fix accesibility issue in AutoFields buttons

### A picture is worth a thousand words

image

https://user-images.githubusercontent.com/6642927/136169649-75eeedd5-e4de-489d-85c9-b1ebb95ce535.mov

### How to replicate

* Go to Applications Menu > Remote Apps
* Create a new Remote App
* Change type to Custom Element
* Inspect +/- buttons and check they now have a title attribute (and/or hold your mouse over the button and check that title appears as a tooltip)

### Checklist

- ~~[ ] I have made corresponding changes to the documentation~~
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked this works in the browser

### CODE REVIEW

File: modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/auto_fields.js [hunk 1]
```diff
@@ -29,12 +29,16 @@ AUI.add(
 		];
 
 		var TPL_ADD_BUTTON =
-			'<button class="add-row btn btn-icon-only btn-monospaced btn-secondary toolbar-first toolbar-item" title="" type="button">' +
+			'<button class="add-row btn btn-icon-only btn-monospaced btn-secondary toolbar-first toolbar-item" title="' +
+			Liferay.Language.get('add') +
```
[the remark this answers is not in the document]
Comment (carloslancha) [reply modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/auto_fields.js:33]: There's no design about this, so for me Add and Remove are fine, but maybe @marcoscv-work have something to say about it

**Comment by carloslancha** [comment]
LNHTM!

Congrats for that amazing description, you've joined the dark side smoothly!

**Review by carloslancha — APPROVED** [review]

**Review by julien — APPROVED** [review]
LGTM

**Comment by tinycarol** [comment]
ci:reevaluate:1337401_331

**Comment by nhpatt** [comment]
You can ignore the failure and send it directly to Brian, as the CI is having problems with Okta integration and the permissions to download the checkoutDXP script are failing (since yesterday).

**Comment by nhpatt** [comment]
First PR merged: https://github.com/brianchandotcom/liferay-portal/pull/108270

🎉 🎉 🎉

**Pull request closed without merging**